src/sign/interactive_sign/participant.rs

```diff
@@ -119,13 +140,33 @@ impl ProtocolParticipant for InteractiveSignParticipant {
         self.presigner.other_ids()
     }
 
-    #[allow(unused)]
     fn process_message<R: RngCore + CryptoRng>(
         &mut self,
         rng: &mut R,
         message: &Message,
```

Comment:
Random question: Is there a reason `process_message` takes the message by ref instead of value?

Reply:
It's \~legacy\~

I'm not even sure if there's a good argument for it because the library handles too-early caching. I don't think you could even use it to manually implement identifiable abort in the case where processing the message fails. I'd be fine to change it to pass-by-value instead; I don't anticipate this breaking anything in our potential use cases.

Reply:
Thanks! Good to know. That's sorta what I figured.